.github/workflows/odis-signer-container.yml

```diff
@@ -0,0 +1,44 @@
+---
+  name: Build ODIS signer image
+  
+  on:
+    push:
+      paths:
+        - 'dockerfiles/phone-number-privacy/Dockerfile-signer'
+        - 'packages/phone-number-privacy/signer/**'
+      branches:
+        - main
+    pull_request:
+      paths:
+        - 'dockerfiles/phone-number-privacy/Dockerfile-signer'
+        - 'packages/phone-number-privacy/signer/**'
+    workflow_dispatch:
+  
+  jobs:
+    odis-signer-build-dev:
+      uses: celo-org/reusable-workflows/.github/workflows/container-cicd.yaml@v1.11.4
```

Comment:
Have you try with `master` latest commit?

Reply:
Didn't have time today no